plugins/woocommerce/src/Blocks/InteractivityComponents/Dropdown.php

```diff
@@ -90,7 +111,24 @@ class="components-button components-form-token-field__remove-token has-icon"
 									</span>
 								<?php } ?>
 							<?php } ?>
-							<input id="components-form-token-input-1" type="text" autocomplete="off" data-wc-bind--placeholder="state.placeholderText" class="components-form-token-field__input" role="combobox" aria-expanded="false" aria-autocomplete="list" aria-describedby="components-form-token-suggestions-howto-1" value="" data-wc-key="input">
+							<style>
+								#<?php echo esc_html( $input_id ); ?>::placeholder {
+									<?php echo esc_attr( $text_color_style ); ?>
+								}
+							</style>
```

Comment:
I think we can use [print_late_styles](https://developer.wordpress.org/reference/functions/print_late_styles/) to avoid doing this. The ID need to be consistent between page loads though.

Reply:
@dinhtungdu I don't think I need to use `print_late_styles` if I just use: [wp_add_inline_style](https://developer.wordpress.org/reference/functions/wp_add_inline_style/) Does that sound right to you?

Reply:
@samueljseay  Ah yeah, you're right! Blocks are rendered before `wp_head` so it's not late style

Reply:
@dinhtungdu This was a great suggestion btw, thank you. Added in https://github.com/woocommerce/woocommerce/pull/43068/commits/b6b900fd94ea05216d93293ee5125ea0b2641861